SparkPost/matchbox
# FE-1007 Manually pick Grid props

### What Changed
- Manually picks grid props – the propeTypes reference was causing a bug seen here: https://nostalgic-stonebraker-a9b0c1.netlify.com/?path=/story/utility-grid--responsive
- `babel-preset-react-app` applies `babel-plugin-transform-react-remove-prop-types`
- Unsure if App production build is affected

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Verify extra props are omitted

### PR Checklist
